opm/input/eclipse/Python/PyRunModule.cpp

```diff
@@ -45,33 +45,18 @@ PyRunModule::PyRunModule(std::shared_ptr<const Python> python, const std::string
     if (!fs::is_regular_file(file))
         throw std::invalid_argument("No such module: " + fname);
 
-    std::string module_name = file.filename().stem();
-    std::string module_path = file.parent_path().string();
-    if (!module_path.empty()) {
-        py::module sys = py::module::import("sys");
-        py::list sys_path = sys.attr("path");
-        {
-            bool have_path = false;
-            for (const auto& elm : sys_path) {
-                const std::string& path_elm = static_cast<py::str>(elm);
-                if (path_elm == module_path)
-                    have_path = true;
-            }
-            if (!have_path)
-                sys_path.append(py::str(module_path));
-        }
+    this->module_path = fs::current_path().string() / file.parent_path();
+    this->module_name = file.filename().stem();
+
+    // opm_embedded needs to be loaded before user defined module.
+    try {
+        this->opm_embedded = py::module::import("opm_embedded");
+    } catch (const std::exception& e) {
+        std::cerr << "Exception thrown when loading Python module opm_embedded: " << e.what() << std::endl;
+        throw e;
+    } catch (...) {
+        std::cerr << "General exception thrown when loading Python module opm_embedded!" << std::endl;
```

Comment:
If this is catched and handled somewhere else, then it might be better to create a new exception with a better error message that indicates that this happend when running PYACTION.

If that is exception is handled elsewhere and logged we do not need any message here. If not we should use something like
```
OpmLog::error(fmt::format("Exception thrown when loading Python module opm_embedded: {}), e.what());
```